### DISCUSSION TITLE
Arguments in functions - naming convention

### DISCUSSION THREAD

**Discussion by JanSzczepan** [Cyfrin/foundry-full-course-cu#518] [Q&A]
Hello everyone, I have been wondering which naming convention should we use in Solidity functions?

```solidity
function setNumber(uint256 number) public {
   s_number = number;
}
```
or
```solidity
function setNumber(uint256 _number) public {
   s_number = _number;
}
```

I also noticed that chainlink uses underscores in mocks (VRFCoordinatorV2Mock) and not in other contracts like VRFCoordinatorV2.

**Comment by Niferu** [comment]
Hello @JanSzczepan 

It's basically up to you, but in this course we are using no underscores in main contracts codes.

**Comment by JanSzczepan** [comment]
Ok, so is there any best practice or can I just use whatever option I like more?

**Reply by Niferu** [reply]
Take a look [here](https://www.geeksforgeeks.org/solidity-style-guide/) :)

As you probably know coding is free space and everyone code looks a bit different, so do not get bother by that I would stick to my own style to be honest. If you want to match some guides just find one on google. Naming is important but even more important is code formatting, so if you are using solidity, javascript etc. I can highly recommend using prettier, so everyone can match their own formatting at least :)

Name should be relevant to function/argument and adding some underscores or not is not that important.
